docs/resources/iam_policy.md

```diff
@@ -0,0 +1,70 @@
+---
+page_title: "Scaleway: scaleway_iam_policy"
+description: |-
+Manages Scaleway IAM Policies.
+---
+
+# scaleway_iam_policy
+
+| WARNING: This resource is in beta version. If your are in the beta group, please set the variable `SCW_ENABLE_BETA=true` in your `env` in order to use this resource. |
+|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+
+Creates and manages Scaleway IAM Policies. For more information, see [the documentation](https://developers.scaleway.com/en/products/iam/api/v1alpha1/#policies-54b8a7).
+
+## Example Usage
+
+```hcl
+resource "scaleway_iam_policy" "main" {
+  name = "my policy"
+  description = "a description"
+  no_principal = true
+  rule {
+    project_ids = ["xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"]
+    permission_set_names = ["AllProductsFullAccess"]
+  }
+}
+```
```

Comment:
Need a more complete example that gives single project object storage read access to an application